apps/examples/wifi_manager/wm_test/wm_test_usage.h

```diff
@@ -94,6 +94,10 @@
 	"\n it can be run after a device is connected to an AP\n"	\
 	"	 wm_test dns count\n"
 
+#define WT_CONNECTBYBESTRSSI_USAGE											\
```

Comment:
please check formatting of preprocessor at all places

Reply:
Done